src/custom-element.js

```diff
@@ -1362,8 +1362,9 @@ function createBaseCustomElementClass(win) {
      * @package @final @this {!Element}
      */
     getRealChildren() {
-      return dom.childElements(this, element =>
-        !isInternalOrServiceNode(element));
+      return dom.childElementByTag(this, '*').filter(element => {
```

Comment:
I was after code size here, not speed. I'll split out the `childElements` changes.